### ISSUE TITLE
[ICC][Sindragosa]LoS Exploit last phase (35%)

### ISSUE THREAD

**Issue by Crispybuilt** [dalaranwow/dalaran-wow#4849]
**Issue present:** Once you reach the 35% Phase on any mode of Sindragosa, she will start casting Mystic Buffet (AoE spell which applies a stacking debuff on each individual player)
- The issue is that if a player with the Frost Beacon places a Frost Tomb inside the hitbox of Sindragosa (more or less close to the center of her model) she will stop applying stacks of Mystic Buffet to any player, including the tank.

**How it should work:** Regardless the fact that this is a clever method to get rid of the stacking debuff, this is not a valid method of doing this encounter, hence it should be fixed in a way that Mystic Debuff gets applied to every player if not properly positioned behind the Frost Tomb.

**Video link how to reproduce the bug:** 
*https://youtu.be/mfNbA6yyK2Y?t=6m17s*
*https://youtu.be/mfNbA6yyK2Y?t=7m* (Tank reseting stacks when perfectly positioned inside)

- **IDs:** 
https://wowgaming.altervista.org/aowow/?spell=70127
https://wowgaming.altervista.org/aowow/?spell=70128
https://wowgaming.altervista.org/aowow/?spell=72528
https://wowgaming.altervista.org/aowow/?spell=72529
https://wowgaming.altervista.org/aowow/?spell=72530

https://wowgaming.altervista.org/aowow/?search=frost+tomb#spells

**Edit**- Forgot to mention that Blistering Cold will have no effect pulling anyone with this method and not doing any damage no matter the distance between player/boss.

**Comment by Crispybuilt** [comment]
Merging #4773 
Since this one has valid proof

**Comment by Vandydorias** [comment]
Blistering cold is supposed to be completely negated by the ice blocks regardless of distance. That's how line of sight works.

**Comment by Rushor** [comment]
Game/Spells: Sindragosa - Mystic Buffet - Update LOS mechanic

* closes: https://github.com/dalaranwow/dalaran-wow/issues/4849
* commit: 3b1313868622fa2f516e90e35a47610e43887b0e
* commit: d3444b14286c7f4359253222d368250df52df56a

The issue has been fixed by a developer and will be tested on the Public Test Realm in the future (PTR). The changes/fixes will be applied with the NEXT changelog on the liverealm in the future.
This means that the issue exists in game at the moment i wrote this message.
**Only once the issue was marked with "Fixed - On Live Server", you can find the fix on your game server.**
Please watch http://www.dalaran-wow.com/news/ to see when the changelog and the fix was applied.

**Comment by Crispybuilt** [comment]
@Vandydorias Yes, that is correct, but in this particular case of an exploit, then Blistering Cold has no purpose at all in the last Phase. Which makes the encounter significantly easier.
It is correct it should negate the Pull+Damage if you are in LoS, but this LoS is not working correctly.

----------------

- I'll try to demonstrate in pictures how last phase should look like, retail version.
Like Horvic said in the previous report:
*Check any and every retail video, if this was possible on retail it would have been used as the primary strategy.*

![1](https://user-images.githubusercontent.com/5054924/31985279-d364c352-b964-11e7-8fd8-f634bd3d3793.jpg)
![2](https://user-images.githubusercontent.com/5054924/31985283-d5e67a80-b964-11e7-9be5-86134b0b013e.jpg)
![3](https://user-images.githubusercontent.com/5054924/31985286-d79a2124-b964-11e7-9e96-5e6ab5dc54c8.jpg)
![4](https://user-images.githubusercontent.com/5054924/31985289-d9aab62c-b964-11e7-9127-bacdd82e3730.jpg)
![5](https://user-images.githubusercontent.com/5054924/31985291-dbeefe0c-b964-11e7-842c-9e4d9ab97994.jpg)
![6](https://user-images.githubusercontent.com/5054924/31985312-f1ee4050-b964-11e7-8304-79e924512356.jpg)
![7](https://user-images.githubusercontent.com/5054924/31985316-f4b9a5ae-b964-11e7-84bf-0fbf830b0195.jpg)
![8](https://user-images.githubusercontent.com/5054924/31985320-f63c4d28-b964-11e7-93ce-2415b369cfeb.jpg)
![9](https://user-images.githubusercontent.com/5054924/31985323-f7f58dfa-b964-11e7-9ffd-36d37da5f35f.jpg)

And repeat.

What guilds are currently doing on Algalon:
![1a](https://user-images.githubusercontent.com/5054924/31985696-406e8ea0-b966-11e7-9324-10856d9ff3b6.jpg)
![1b](https://user-images.githubusercontent.com/5054924/31985698-41f505d8-b966-11e7-9e42-1a9a171776e9.jpg)
![1c](https://user-images.githubusercontent.com/5054924/31985700-438d7f10-b966-11e7-8f34-a2dc8040424a.jpg)
![1d](https://user-images.githubusercontent.com/5054924/31985823-a10a9740-b966-11e7-8310-4ce09d47e134.jpg)

**Comment by TeridanDW** [comment]
Hello, me and @Crispybuilt have tested this on PTR and the result is the same as pointed above:
https://user-images.githubusercontent.com/5054924/31985823-a10a9740-b966-11e7-8310-4ce09d47e134.jpg

**Comment by Crispybuilt** [comment]
In addition to that, I will copy what @Horvic said in #4773 

> - **You should either be in or out of line of sight of Sindragosa never both at the same time. **
> - **If the Ice Tomb gets placed too close to Sindragosa the entire raid should still get Mystic Buffet as the Tomb is unable to provide adequate line of sight.**

**Steps how the fix should look:**
- Player has to be directly behind the Ice Tomb to Line of Sight Sindragosa
- If the player is more than 10 yards away, behind the Ice Tomb, ignore Line of Sight (Apply Mystic Buffet 
stacks + Blistering Cold shall pull him)
- Do not allow player to use any spells on Sindragosa/Players if in the Line of Sight of Sindragosa no matter the distance

**Comment by Crispybuilt** [comment]
**Additional info about the Frost Tombs:**
- _Ice Tombs can not be Charged/Intercepted. They should be

**Comment by datbunneh** [comment]
@Crispybuilt "Do not allow player to use any spells on Sindragosa/Players if in the Line of Sight of Sindragosa no matter the distance" - I guess you meant when out of LoS and healers should still be able to heal people behind the tomb.
